Makefile

```diff
@@ -0,0 +1,20 @@
+VERSION := $(shell git tag | grep ^v | sort -V | tail -n 1)
```

Comment:
Maybe `git describe --tags` is better? That will show the current tag on the current branch - which means you can do things like:

```
$ git checkout v1.0.1
$ make
```

If I read correctly, the above will always return the latest version in the git repository, unrelated to what commit you are building from.